event/handler.go

```diff
@@ -289,10 +298,10 @@ func (handler *ExportHandler) fullDownload(ctx context.Context, event *FilterSub
 		"version":     event.Version,
 	})
 
-	fileID := uuid.NewV4().String()
-	log.Event(ctx, "storing pre-publish file", log.INFO, log.Data{"fileID": fileID})
+	name := event.DatasetID + "-" + event.Edition + "-v" + event.Version
```

Comment:
Is it possible that we want to generate more than one file with the same datasetID, Edition and Version? In this case we would have a name collision. 
I imagine that files will be unique by {DatasetID, Edition, Version}, but just wanted to double check.

Reply:
I like the question, but no there shouldn't be a name clash as the full dataset should only be run once. However if there was an issue in processing and the file was uploaded to s3 but the version document failed to update the status to `published` then we may be backed into a corner as this function is not idempotent?

I will double check the `handler.fileStore.PutFile` and see if it overwrites an existing file if the name clashes

Reply:
I've double checked and the s3 function we use to upload file to s3 does overwrite an existing file that has the same file location and name. Tested this by hardcoding the name of the file in both dp-dataset-exporter and dp-dataset-exporter-xlsx services.

Im still happy with this!

Reply:
Thanks! 👍